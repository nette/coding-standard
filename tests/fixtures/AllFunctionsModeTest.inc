<?php // {"optimizedFunctionsOnly": false}

namespace App;

use const E_ALL;
use function strlen;
use function count;
use const PHP_OS;

class MyClass
{
    public function run()
    {
        \strlen('test');
        $c = \count([]);
        $v = \PHP_VERSION;
        $os = PHP_OS;
    }
}
