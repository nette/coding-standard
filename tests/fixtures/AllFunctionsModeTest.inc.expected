<?php // {"optimizedFunctionsOnly": false}

namespace App;

use const PHP_OS, PHP_VERSION;
use function count, strlen;

class MyClass
{
    public function run()
    {
        strlen('test');
        $c = count([]);
        $v = PHP_VERSION;
        $os = PHP_OS;
    }
}
